### PR TITLE
Add PR template

### PR DESCRIPTION
**Pull request by caesarxuchao**
to instruct people to submit PRs to the main repository for now.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<!--  Sorry, client-go does not accept pull requests yet. Please submit your
```
Comment (lavalamp) [inline .github/PULL_REQUEST_TEMPLATE.md]: Does it have to be in an html comment?

s/accept pull requests yet/accept changes via pull requests at this time/

**Comment by lavalamp** [comment]
lgtm with one minor wording change

**Comment by caesarxuchao** [comment]
Done. PTAL. Thanks.

**Comment by lavalamp** [comment]
LGTM, thanks!

**Pull request merged**